wordpress-mobile/WordPress-iOS
# History / Revisions: Revisions list UI

Refs. #10304 

![revisions-list-ui](https://user-images.githubusercontent.com/912252/48257530-aa229880-e40a-11e8-8f3c-587fd29df426.jpg)

This PR introduces the UI for the revisions list (table view and cell)

Modified / Created elements:
- The `UITableView` and headers are aligned with our style guide. The header is the same class used by Pages and Activity Log.
- `RevisionsTableViewCell` includes all the element we have to display, like title, subtitle, image and stack view for the revision operation view.
- `RevisionOperation` is a class and owner of `RevisionOperationView`. It uses the same logic as the `AlertView` to load and display the view using the nib file. `RevisionOperationView` is the view that display the revision operation data.
- `RevisionsTableViewFooter` is the table view footer.

**To test:**
- Open a post with some revisions, ensure the history button appears and shows a list of revisions
- Revisions should appear on the list with the new layout applied

**Note:**
This PR doesn't include the author informations


@iamthomasbishop this is the last screenshot with the final result:

![simulator screen shot - iphone x - 2018-11-06 at 20 22 45](https://user-images.githubusercontent.com/912252/48257571-c1618600-e40a-11e8-83f8-2047e6559466.png)
